WikiHowAgent.py

```diff
@@ -1,47 +1,54 @@
 import spacy
 from pywikihow import WikiHow, search_wikihow
 
+
 def Question_generate(content):
-  verb  = []
+  verb = []
   noun = []
   nlp = spacy.load("en_core_web_sm")
   pos = content.lower().find("!howdoi")
-  Q_Token = nlp(content[pos+len("!howdoi"):])
-  for token in Q_Token :
+  sentence = "how to" + (content[pos+len("!howdoi"):])
+  Q_Token = nlp(sentence)
+  for token in Q_Token:
     print(token.pos_)
-    if token.pos_ in ["VERB" ,"NOUN","PROPN"] :
+    if token.pos_ in ["VERB", "NOUN", "PROPN"]:
       verb.append(token)
-  if len(verb) !=0:
+  print(verb)
+  if len(verb) != 0:
       v_pos = content.lower().find(verb[0].text)
       QUESTION = "how to "+content[v_pos:]
```

Comment:
maybe this can be modified to QUESTION = sentence or something else that works since we already do similar things up there anyway